### ISSUE TITLE
weight decay

### ISSUE THREAD

**Issue by HHHHStar** [HuiZhang0812/CreatiLayout#18]
Great Work! I am wondering the hyperparameter of weight decay.
